node/pkg/fetcher/types.go

```diff
@@ -20,6 +20,8 @@ const (
 	InsertLocalAggregateQuery             = `INSERT INTO local_aggregates (config_id, value) VALUES (@config_id, @value)`
 	DECIMALS                              = 8
 	DefaultStreamInterval                 = time.Second * 5
+	ForeignExchangePricePairs             = "GBP-USD,EUR-USD,KRW-USD,JPY-USD,CHF-USD"
```

Comment:
@nick-bisonai Can we have the definition of what is FX pair inside of config json rather than inside of code? 🙏

Reply:
do you think I should add more property inside config object?

Reply:
Yes, that would be nice. We would not have to update code when new FX pair is added. Less hidden settings is good to have.

Reply:
it requires lot of code updates, I'll make another pr later for this